crates/bevy_sprite/Cargo.toml

```diff
@@ -24,9 +25,9 @@ bevy_math = { path = "../bevy_math", version = "0.17.0-dev" }
 bevy_picking = { path = "../bevy_picking", version = "0.17.0-dev", optional = true }
 bevy_reflect = { path = "../bevy_reflect", version = "0.17.0-dev" }
 bevy_transform = { path = "../bevy_transform", version = "0.17.0-dev" }
-bevy_window = { path = "../bevy_window", version = "0.17.0-dev" }
+bevy_window = { path = "../bevy_window", version = "0.17.0-dev", optional = true }
```

Comment:
bevy_camera hard-depends on bevy_window, so this isn't actually doing anything

Reply:
But removing direct dependencies or making them optional is still preferable I think, or don't you think so?
It seems like it clarifies things to me and makes it more explicit why and where we are using each dependency.

Reply:
sounds good to me

Reply:
Yup this can be sorted out, theres value in being explicit here.